libsql_client/hrana/client.py

```diff
@@ -26,6 +26,15 @@
 
 def _create_hrana_client(config: _Config) -> HranaClient:
     assert config.scheme in ("ws", "wss")
+    if config.scheme == "ws" and config.tls:
+        raise LibsqlError(
+            "A 'ws:' URL cannot opt into TLS by using ?tls=1", "URL_INVALID"
+        )
+    elif config.scheme == "wss" and not config.tls:
+        raise LibsqlError(
+            "A 'wss:' URL cannot opt out of TLS by using ?tls=0", "URL_INVALID"
+        )
```

Comment:
this is replicated with dbapi2, if moved into HranaConn, it would be the same in both cases

Reply:
Fixed